crates/aptos-crypto/src/arkworks/shamir.rs

```diff
@@ -112,6 +111,52 @@ fn naive_all_lagrange_coefficients<F: Field>(xs: &HashSet<F>) -> Vec<(F, F)> {
     results
 }
 
+// TODO: maybe move this elsewhere?
+/// Computes the Lagrange denominators for a set of evaluation points in a Radix-2 FFT domain.
+///
+/// Specifically, for a polynomial evaluated at points `\omega^0, \dots, \omega^{n-1}` in `dom`,
+/// the Lagrange denominators are given by:
+///
+/// ```text
+/// v_i = 1 / \prod_{j \ne i} (\omega^i - \omega^j)
+/// ```
+#[allow(non_snake_case)]
+pub fn all_lagrange_denominators<F: FftField>(
```

Comment:
This is a copy-paste-and-modify-to-arkworks from aptos-dkg? Any other changes to it?